components/harvest/actions/start-timer/start-timer.mjs

```diff
@@ -0,0 +1,29 @@
+import harvest from "../../harvest.app.mjs";
+
+export default {
+  key: "harvest-start-timer",
+  name: "Start Time Entry",
+  description: "Restart a stopped timer entry. [See docs here](https://help.getharvest.com/api-v2/timesheets-api/timesheets/time-entries/)",
```

Comment:
I guess this doc link needs to be updated

Reply:
Not really.
Time entry related api docs reside on same link.